packages/evm-predicates/scripts/versions.ts

```diff
@@ -4,5 +4,5 @@ import { generateVersions } from '@fuel-connectors/common/scripts';
 
 const __filename = fileURLToPath(import.meta.url);
 const currentDir = path.dirname(__filename);
-const __dirname = path.normalize(`${currentDir}/../src/generated/predicates`);
+const __dirname = path.normalize(`${currentDir}/../src/predicates`);
```

Comment:
keep the `generated` folder as it makes clear for devs that it's code generated by some script and should not be edited directly

Reply:
https://github.com/FuelLabs/fuel-connectors/pull/235/commits/d44132687002b4f8d6332e351a1c0eefa76387cd